test/Zapper.ts

```diff
@@ -205,8 +342,8 @@ describe("Zapper test suite", function () {
             const tokenUSDC = new ethers.Contract(usdcAddress, abiUSDTToken, owner);
             const usdcBalance = await tokenUSDC.balanceOf(owner.address);
 
-            await zapOutPositionWithAnyBase(r, zapper, tokenUSDC);
-            await printPositionInfo(r);
+            await zapOutPositionWithAnyBase(r, zapper, tokenUSDC, bnFromNum(10, 6));
```

Comment:
What is bnFromNum(10, 6)? Where it is used and where is the method with this  paramater?